packages/host/config/environment.js

```diff
@@ -46,6 +46,12 @@ module.exports = function (environment) {
     resolvedBaseRealmURL:
       process.env.RESOLVED_BASE_REALM_URL || 'http://localhost:4201/base/',
     featureFlags: {},
+    stripeCustomerPortalLink:
+      process.env.STRIPE_CUSTOMER_PORTAL_LINK ||
+      'https://billing.stripe.com/p/login/test_cN216h3BlbML3FS144',
+    stripeApiKey:
+      process.env.STRIPE_API_KEY ||
+      'sk_test_51Q8hkbPUHhctoJxanM9uVZban3mK5he2g1s7r1gKvBXTpejGxUdJHSjWLlmB8LzIaffiQj9Bj96ag0OxTwNl5BGk00Ov4gjcBC',
```

Comment:
Never put API keys to version control. I don't know which Stripe account this key is for, but if it's Cardstack's you need disable and rotate this key in Stripe and make sure it gets applied to our infra.

Reply:
Sorry my bad, it's the sandbox api key. This what I asked you yesterday about where to put the api key to fetch payment links. Is it better to use each payment link instead?

Reply:
Related discussion: https://github.com/cardstack/boxel/pull/1777#discussion_r1858142371